src/views/Lists.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <b-container>
+    <Title msg="Lists" />
+    <ListTable />
+  </b-container>
+</template>
+
+<script>
+import ListTable from "@/components/lists/ListTable";
+import Title from "@/components/HelloWorld";
+
+export default {
+  name: "Lists",
+  data() {},
```

Comment:
Data has to return a JS object.  It looks like this does nothing so it's probably safe to remove this line.
```suggestion
```